tools/scripts/setup_helper.py

```
@@ -0,0 +1,194 @@
+"""
+This script provides helper functions for setting up the development environment,
+managing contexts, and handling service architecture operations.
+"""
+
+import argparse
+import json
+
+import subprocess
+import sys
+from pathlib import Path
+import shutil
```

Ruff (0.12.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)